packages/server/src/auth/auth.service.ts

```diff
@@ -25,6 +27,22 @@ export class AuthService {
     );
   }
 
+  async createStudentSubscriptions(userId: number): Promise<void> {
+    const memberMailServices = await MailServiceModel.find({
+      where: { mailType: 'member' },
+    });
+
+    const subscriptions = memberMailServices.map((service) => {
+      const subscription = new UserSubscriptionModel();
+      subscription.userId = userId;
+      subscription.serviceId = service.id;
+      subscription.isSubscribed = true;
+      return subscription;
+    });
+
+    await UserSubscriptionModel.save(subscriptions);
```

Comment:
might want to put a try-catch either around this .save or around anything that calls createStudentSubscription() (I suggest the latter since it catches anything with the .find as well)